### PR TITLE
Write documentation for posting a new job listing

### PR DESCRIPTION
**Pull request by Dalek-Sec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ We recommend using [Atom](https://atom.io). You can use their GUI (graphical use
 Get repository locally by running
 `git clone https://github.com/trysparrow/sparrow-landing.git`
 
-<set user information>
+TODO: set user information
```
Comment (dhanus) [inline README.md]: Is this a todo for you or someone else?

**Comment by Dalek-Sec** [comment]
resolves #41

File: README.md [hunk 2]
```diff
@@ -87,6 +88,31 @@ The blank line between front matter and content is required.
...
+title: <Page Title (can match job title)>
+job_title: <Job Title>
+job_location: <City, Country>
+banner: "<filename in path /assets/images/banners>"
+description: "<One sentence catch for job description>"
+---
+
+```
+#### Content
+Jekyll posts can be written in a mix of Markdown and HTML.
```
Comment (dhanus) [inline README.md]: I might just put the syntax link here. 

Jekyll posts can be written in a mix of [Markdown](link to guide) and [HTML](link to some not bad HTML cheat sheet)

File: README.md [hunk 3]
```diff
@@ -87,6 +88,31 @@ The blank line between front matter and content is required.
...
+[Here](https://www.markdownguide.org/basic-syntax) is a guide for Markdown syntax.
+#### Posting on careers page
+Minimal understanding of HTML is required in this part.
```
Comment (dhanus) [inline README.md]: Change to "To do this part, you will need to understand a bit of HTML"

File: README.md (hunk 1 above)
Comment (Dalek-Sec) [reply README.md]: Me

**Pull request merged**